src/OpenLoco/Industry.cpp

```diff
@@ -13,6 +13,18 @@ using namespace OpenLoco::Map;
 
 namespace OpenLoco
 {
+    static const map_pos3 word_4F9274[] = {
```

Comment:
these aren't map_pos3's they are map_pos with some other variable in the third position that is used with `station_elements->_0 & 0xC0`